src/Runtime/OMTensor.inc

```diff
@@ -192,11 +222,10 @@ OMTensor *omTensorCreateEmpty(
 
 /* OMTensor destroyer */
 void omTensorDestroy(OMTensor *tensor) {
-  if (tensor->_owning) {
+  if (tensor->_owning)
     free(tensor->_allocatedPtr);
```

Comment:
Might still want to set allocated/aligned ptrs to NULL? This gives us a way to check whether these pointers have been initialized (i.e., initialized if and only if not NULL).

Reply:
Setting those pointers to NULL is basically a "waste" since once the `tensor` is freed, its content is considered garbage, i.e., whatever we are setting immediately becomes garbage and nobody should be looking at the content of a freed `tensor` structure. Setting them to NULL is only useful if we intend to reuse the `tensor` structure later on but that's not the case here.